platform/ios/src/MGLMapView.h

```diff
@@ -79,6 +79,21 @@ typedef NS_ENUM(NSUInteger, MGLUserTrackingMode) {
     MGLUserTrackingModeFollowWithCourse,
 };
 
+/** Options for `MGLMapView.preferredFramesPerSecond`. */
+typedef NSInteger MGLMapViewPreferredFramesPerSecond NS_TYPED_EXTENSIBLE_ENUM;
```

Comment:
This is implemented with [`NS_TYPED_EXTENSIBLE_ENUM`](https://developer.apple.com/documentation/swift/objective_c_and_c_code_customization/grouping_related_objective_c_constants) (instead of using `NS_ENUM` or the like) to enable Swift users to omit the full key name and just use the type-aware dot syntax (e.g., `mapView.preferredFramesPerSecond = .lowPower`).